scripts/sysdig-deploy/update-sysdig-deploy.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+set -e
+
+node_analyzer_chart_path="node-analyzer"
+agent_chart_path="agent"
+kspm_collectore_chart_path="kspm-collector"
+rapid_response_chart_path="rapid-response"
+admission_controller_chart_path="admission-controller"
+sysdig_deploy_path="sysdig-deploy"
+
+minor=0
+major=0
+patch=0
+
+check_update_needed () {
+    
+    new_subchart_version=$(yq eval '.version' charts/$chart/Chart.yaml)
+    sysdig_subchart_version=$(yq eval '.dependencies[] | select(.name == "'$chart'") | .version ' charts/$sysdig_deploy_path/Chart.yaml | sed "s/~//")
+
+    echo "subchart version in" $chart"/Chart.yaml"
```

Comment:
I believe this should do the trick (but please correct me if I'm wrong!)
```suggestion
    echo "subchart version in ${chart}/Chart.yaml"
```